src/index.js

```diff
@@ -8,7 +8,7 @@ import Dropdown from './dropdown';
 import Header from './header';
 import Footer from './footer';
 
-export default {
+const Oxygen = {
```

Comment:
How do we consume this on the "client" side?

Reply:
Exactly the same as you would, nothing changed other than it being a named export.

Reply:
Oh right!